src/storages/types.ts

```diff
@@ -118,16 +126,52 @@ export interface ICustomStorageWrapper {
    * The promise rejects if the operation fails, for example, if there is a connectin error or the key holds a value that is not a list.
    */
   getItemsCount: (key: string) => Promise<number>
+
+  /** Set operations */
+
   /**
-   * Returns if item is a member of a list.
+   * Returns if item is a member of a set.
    *
    * @function itemContains
-   * @param {string} key list key
+   * @param {string} key set key
    * @param {string} item item value
-   * @returns {Promise<boolean>} A promise that resolves with true boolean value if `item` is a member of the list stored at `key`, or false if it is not a member or `key` list does not exist.
-   * The promise rejects if the operation fails, for example, if there is a connectin error or the key holds a value that is not a list.
+   * @returns {Promise<boolean>} A promise that resolves with true boolean value if `item` is a member of the set stored at `key`, or false if it is not a member or `key` set does not exist.
+   * The promise rejects if the operation fails, for example, if there is a connectin error or the key holds a value that is not a set.
    */
   itemContains: (key: string, item: string) => Promise<boolean>
+  /**
+   * Add the specified `items` to the set stored at `key`. Those items that are already part of the set are ignored.
+   * If key does not exist, an empty set is created before adding the items.
+   *
+   * @function addItems
+   * @param {string} key set key
```

Comment:
(Don't hate me again :P) I would start the param description with Upper case, and end it with period.
Ex:
`   * @param {string} key  Set key.`

Reply:
Good point Emi. I would fix them now, and take it into account next time.